### PR TITLE
Adjacency Matrix test coverage increased by adding test line

### PR DESCRIPTION
**Pull request by akshatahuja1**
#### Brief description of what is fixed or changed

Increased the test coverage for the adjacency matrix implementation of graph data structure, by adding tests corresponding to the add_edge method.

### CODE REVIEW

File: pydatastructs/graphs/tests/test_adjacency_matrix.py [hunk 1]
```diff
@@ -26,6 +26,5 @@ def test_AdjacencyMatrix():
     assert neighbors == [v_1]
     g.remove_edge(0, 1)
     assert g.is_adjacent(0, 1) is False
-
-    assert raises(ValueError, lambda: g.add_edge('u', 'v'))
-    assert raises(ValueError, lambda: g.add_edge('v', 'x'))
```
Comment (czgdp1807) [inline pydatastructs/graphs/tests/test_adjacency_matrix.py:31]: Why did you remove these tests?

Comment (akshatahuja1) [reply pydatastructs/graphs/tests/test_adjacency_matrix.py:31]: I thought that these tests were in a form that had to be edited (in accordance with the other tests I saw in the tests.py file), which I tried to accomplish by my tests.

**Comment by czgdp1807** [comment]
Codecov shows that the changes are decreasing the coverage.

**Comment by akshatahuja1** [comment]
I made the changes by restoring the previous tests. I have pushed those changes with the commit titled 'Tests added for adjacency matrix'.

**Comment by czgdp1807** [comment]
Thanks @akshatahuja1
